server/apps/frontend/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Extreme carpaccio</title>
+    <meta name="theme-color" content="#000000" />
+
+    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
+    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
+    <!--[if lt IE 9] <div id="root"></div>>
+      <script src="./components/html5shiv/dist/html5shiv.min.js"></script>
+      <script src="./components/respond/dest/respond.min.js"></script>
+    <![endif]-->
+  </head>
+  <body>
+    <nav class="navbar navbar-dark bg-dark fixed-top">
+      <div class="container">
+        <div class="navbar-header">
+          <a class="navbar-brand mb-1 h6" href="#">Extreme carpaccio</a>
+        </div>
+      </div>
+    </nav>
+    <div id="root"></div>
+    <script type="module" src="/src/main.tsx"></script>
+  </body>
+</html>
```

Comment:
You still use the template from Create React App. The one from Vite is enough and simpler.

```suggestion
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Extreme carpaccio</title>
  </head>
  <body>
    <nav class="navbar navbar-dark bg-dark fixed-top">
      <div class="container">
        <div class="navbar-header">
          <a class="navbar-brand mb-1 h6" href="#">Extreme carpaccio</a>
        </div>
      </div>
    </nav>
    <div id="root"></div>
    <script type="module" src="/src/main.tsx"></script>
  </body>
</html>
```